src/System.Collections/tests/Generic/List/List.Generic.Tests.cs

```diff
@@ -52,5 +52,14 @@ protected void VerifyList(List<T> list, List<T> expectedItems)
         }
 
         #endregion
+
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
+        public void CopyTo_ArgumentValidity(int count)
+        {
+            List<T> list = GenericListFactory(count);
+            Assert.Throws<ArgumentException>(() => list.CopyTo(0, new T[0], 0, count + 1));
+            Assert.Throws<ArgumentException>(() => list.CopyTo(count, new T[0], 0, 1));
+        }
```

Comment:
There aren't any other [Fact]s or [Theory]s on this class... seems like it's purely a bunch of helpers.  Is this the right place for this?